ml-agents/mlagents/trainers/sac/optimizer.py

```diff
@@ -171,15 +171,14 @@ def _create_inputs_and_outputs(self) -> None:
         self.visual_in = self.policy.visual_in
         self.next_vector_in = self.target_network.vector_in
         self.next_visual_in = self.target_network.visual_in
-        self.action_holder = self.policy.action_holder
         self.sequence_length_ph = self.policy.sequence_length_ph
         self.next_sequence_length_ph = self.target_network.sequence_length_ph
         if not self.policy.use_continuous_act:
             self.action_masks = self.policy_network.action_masks
         else:
             self.output_pre = self.policy_network.output_pre
 
-        # Don't use value estimate during inference. TODO: Check why PPO uses value_estimate in inference.
```

Comment:
Is this TODO still todo ?

Reply:
Nope, I removed the value estimate during inference in the previous PR.